docs/source/installation.rst

```diff
@@ -17,29 +17,7 @@ new environment by doing:
 Installation methods
 --------------------
 
-a) With ``git`` and ``conda``
```

Comment:
Also, what do you think about recommending `pip install -e .` so that developers don't need to reinstall after every edit.

Reply:
I'd spent a good ~30 minutes yesterday trying to get the conda installation working again, and it was continuously failing to install. I'll try again today. I agree with the -e comment though

Reply:
@kevin218, can you try using the environment.yml file in the main branch and see if it works for you (but maybe first edit the top line of the file to not overwrite your current conda environment)? For me it tries to install the pip dependencies for a really long time and then fails at installing one or the other of the github dependencies, but I don't know if that's a NASA VPN thing or what. I've tried changing the formatting of the git dependency lines in several different ways, but none have worked

Reply:
I used the env.yml file earlier this week to install eureka on a new Ubuntu machine. It was missing celerite, but I fixed that.  Sounds like it might be a VPN issue.